packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.util.ts

```diff
@@ -613,40 +613,34 @@ describe('internal-plugin-metrics', () => {
     [
       {
         signalingState: 'have-local-offer',
-        iceConnectionState: 'connected',
+        iceConnected: false,
```

Comment:
in the diff, this was `connected`, so shouldn't this be `true`?

Reply:
This was off-topic change requested by Marcin. He pointed out that there is no way for webrtc to be in signalingState = 'have-local-offer', and to be iceConnectionState = 'connected'.